src/modules/pos/validator/components/StakeStatus/statusMessages.js

```diff
@@ -7,13 +7,13 @@ import styles from './styles.css';
 
 const unlockTime = 5;
 
-const LiskAmountFormatted = ({ val }) => (
+const LiskAmountFormatted = ({ val, token }) => (
   <span className={styles.subHeadingBold}>
-    <TokenAmount isLsk val={val} />
+    <TokenAmount isLsk val={val} token={token} />
```

Comment:
`isLsk` this could be a bug? what if collecti or other sidechain uses pos consensus and token wont be LSK? since you are passing `token` explicitly it should be handled dynamically?